.cicd/pipeline.yml

```diff
@@ -435,6 +435,14 @@ steps:
     timeout: 20
     skip: ${SKIP_MACOS_10_15}${SKIP_PACKAGE_BUILDER}
 
+  - wait
+  
+  - label: ":docker::ubuntu: Docker - Build 18.04 Docker Image"
+    command: "./.cicd/create-docker-from-binary.sh"
+    agents:
+      queue: "automation-eks-eos-builder-fleet"
```

Comment:
The `automation-eks-eos-builder-fleet` is a beefier fleet than what we need to run aptitude. Let's switch to the `automation-eks-eos-tester-fleet`, which is a lighter instance type.

Reply:
Done.